webdriver/tests/user_prompts/send_alert_text.py

```diff
@@ -0,0 +1,80 @@
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def send_alert_text(session, body=None):
+    return session.transport.send("POST", "session/{session_id}/alert/text"
+                                  .format(session_id=session.session_id), body)
+
+
+# 18.4 Send Alert Text
+
+def test_non_string_input(session):
+    # 18.4 step 2
+    session.url = inline("<p>This is not an editable paragraph.")
+    element = session.find.css("p", all=False)
+    arr = [1, 2, 3, 4, 5]
+    elements = {"null": None,
+                "object": element,
+                "array": arr,
+                "number": 123}
+    session.execute_script("window.result = window.prompt(\"Enter Your Name: \", \"Name\");")
+    for item in elements:
+        body = {"text": elements[item]}
```

Comment:
This iterator is not very Pythonic. Can you change the test to use [`@pytest.parametrize`](https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize)?

This would look roughly something like this:

```py
@pytest.parametrize("text", [None, {}, [], 42, True])
def test_invalid_input(session, text):
    session.execute_script("""window.result = window.prompt("foo")""")
    response = send_alert_text(session, {"text": text})
    assert_error(response, "invalid argument")
```

Note that I haven’t run the above code.